### ISSUE TITLE
Traits+ section is doubling existing traits on the base monster 

### ISSUE THREAD

**Issue by Darthfett** [javalent/fantasy-statblocks#224]
### Check for existing bug reports before submitting.

- [X] I searched for existing [Bug Reports](https://github.com/javalent/fantasy-statblocks/issues?q=is%3Aopen+label%3Abug+sort%3Aupdated-desc) and found no similar reports.

### Expected Behavior

When using traits+, I expect new traits to be added to the monster's existing traits.

### Current behaviour

New traits are added, but the existing traits are showing up twice.

### Reproduction

Trampling charge is showing up twice:

```statblock
monster: Gorgon
traits+:
  - name: "Boss"
    desc: "As a Boss monster, this creature acts twice (action + bonus action) at Initiative 15 and 5, and performs a basic attack or movement at Initiative 10 and 0.  This creature still only receives one reaction per round."
```

### Which Operating Systems are you using?

- [ ] Android
- [ ] iPhone/iPad
- [ ] Linux
- [ ] macOS
- [X] Windows

### Obsidian Version Check

1.2.8

### Fantasy Statblocks Version

3.6.5

### Confirmation

- [X] I have disabled all other plugins and the issue still persists.

**Comment by Darthfett** [comment]
See attached image, shows that `Trampling Charge` is repeated twice:

![Gorgon](https://github.com/javalent/fantasy-statblocks/assets/412436/33ce937e-dead-4b18-9d3c-72cea80d4e23)

**Comment by sigrunixia** [comment]
Traits are slated to get reworked because of this. Javalent goes on vacation tomorrow for a few days. So we'll suggest creating new code block monsters in the interim until we can nail down this traits issue. 

Currently, implementing the fix for this would re-open #222, so we have some head butting of the code to do.:)

**Comment by Azurewren** [comment]
This is still a problem, can it be reopened?

**Comment by sigrunixia** [comment]
This should be resolved in #249. Note, that PF2E Statblocks are still subject to traits issues for a different reason.
